src/bundle/Resources/views/admin/content_type/create.html.twig

```diff
@@ -6,7 +6,7 @@
 
 {% block _ezrepoforms_contenttype_update_removeFieldDefinition_widget %}
     <button type="{{ type|default('button') }}" {{ block('button_attributes') }}>
-        <svg class="ez-icon ez-icon-trash">
+        <svg class="ez-icon ez-icon--medium ez-icon--light ez-icon-trash">
```

Comment:
Is this class: `ez-icon-trash` used anywhere in CSS or JS?

Reply:
Actually it is not used, I will remove it.